adafruit_tpa2016.py

```diff
@@ -32,6 +32,12 @@
 from adafruit_register.i2c_bits import RWBits
 from adafruit_register.i2c_bit import RWBit
 
+try:
+    import typing  # pylint: disable=unused-import
```

Comment:
Does the `typing` module actually need to be imported? Other than the `busio.I2C` type hint, all the other type hints refer to built-in Python types, so do we actually need anything from the `typing` module?

Reply:
it's not used by the code specifically. But it serves the purpose of raising the exception to prevent the imports that come after it from happening on the microcontrollers.